geth/log/log.go

```diff
@@ -2,16 +2,28 @@ package log
 
 import (
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/ethereum/go-ethereum/log"
 )
 
-// logger is package scope instance of log.Logger
-var logger = log.New("geth", "StatusIM")
+// Logger is a wrapper around log.Logger.
+type Logger struct {
+	log.Logger
+	Level   log.Lvl
+	Handler log.Handler
+}
+
+// logger is package scope instance of Logger
+var logger = Logger{
+	Logger:  log.New("geth", "StatusIM"),
+	Level:   log.LvlInfo,
+	Handler: log.StdoutHandler,
+}
 
 func init() {
```

Comment:
Can we reduce usage of `init()` ?
In `main.go` there is a func `app.Before = func(ctx *cli.Context) error` which we can use to init a logging with flags.